my-dsl.md

```diff
@@ -2,37 +2,78 @@
 _What is the name of the language? Link the name to its webpage 
 (if appropriate)._
 
+I looked at the language [Cg][1].
+
+[1]: https://developer.nvidia.com/cg-toolkit
+
 
 # Domain
 _Describe the language's domain in five words._
 
+vertex/pixel shading for GPUs
+
 
 # Computational model
 _What is the underlying computational model of this language? To answer this 
 question, provide a high-level description (no more than 100 words) of the 
 computation that occurs when someone executes a program in this language._
 
+According to the [Cg tutorial][2], section 1.4.2, Cg is executed to interact
+with the OpenGL or Direct3D API for your GPU. The way this works is for a Cg
+program to be translated into OpenGL or Direct3D, then for the translated
+program to be sent to the GPU using Open GL or Direct3D, and finally for OpenGL
+or Direct3D to execute it on the GPU.
+
+[2]: http://http.developer.nvidia.com/CgTutorial/cg_tutorial_chapter01.html
+
 
 # DSL-ness
 _Fowler writes about a spectrum of languages, from general-purpose languages to 
 "purely" domain-specific. Where does the DSL you chose fall on this spectrum, 
-and why?_ 
+and why?_
+
+Cg is somewhere in the middle. It is intended to be used for GPU operations,
+but it includes so much of C that it could be used more generally. As Fowler
+says on page 30, "A particular usage of a language can put it on either side of
+the DSL line." I would lean towards Cg being more domain-specific, simply
+because there is little reason to use it instead of C unless you are doing GPU
+operations. See ["Cg in Two Pages"][3] for a summary of differences with C.
+
+[3]: http://arxiv.org/pdf/cs/0302013v1.pdf
```

Comment:
I agree with your analysis! One thing I might add is that Cg doesn't support pointers or bitwise operations (from C) or classes, templates, overloading, and other OOP stuff (from C++). This would provide a more concrete reason as to why it is a DSL apart from just the use case.